lib/normalizers/v1.js

```diff
@@ -41,15 +52,26 @@ _.assign(Builders.prototype, {
      */
     auth: function (entityV1) {
         if (!entityV1) { return; }
+        // Handle the noauth/normal case for new and legacy attributes first, in that order
         if (util.notLegacy(entityV1, 'auth') && entityV1.auth) {
             return (entityV1.auth.type === 'noauth') ? { type: 'noauth' } : util.sanitizeAuthArray(entityV1);
         }
-        if (!entityV1.currentHelper) { return; }
+        if (entityV1.currentHelper === 'normal') { return { type: 'noauth' }; }
```

Comment:
By reversing these two conditions, you can get rid of the `util.notLegacy` call.

Reply:
Will take a look. 👍